nixos/lib/make-system-tarball.nix

```diff
@@ -21,14 +21,23 @@
 
   # Extra tar arguments
 , extraArgs ? ""
+  # Command used for compression
+, compressionCommand ? "pixz"
+  # Extension for the compressed tarball
+, compressionExtension ? ".xz"
+  # extra inputs, like the compressor to use
+, extraInputs ? [ pixz ]
+  # flags passed to the compressor
+, compressorFlags ? ""
```

Comment:
Maybe drop compressorFlags and just pass `yourcmd -abc` as the command, instead of as a separate parameter. This simplifies the calling code and we don't do anything special at all with the arguments anyway. Beyond this, which is a minor nit, this LGTM.